### PR TITLE
Remove repetition in `install.sh` by using heredoc

### LINKED ISSUE #104: Remove repetition in `install.sh` by using heredoc
https://linuxize.com/post/bash-heredoc/

### PR DESCRIPTION
**Pull request by ArthurPV**
Close #104

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -95,31 +99,34 @@ fi
...
+EOF
 }
 
 ask_permission_to_install_homebrew_package()
 {
   echo ""
   if [ ! -z ${2+x} ]; then
-    echo "Homebrew installs $1 collectively in one package named '$2'."
-    echo ""
+    echo -e "Homebrew installs $1 collectively in one package named '$2'.\n"
```
Comment (bonachea) [inline install.sh:129]: `echo -e` is non-POSIX and not available when using `/bin/sh` on many versions of macos, which currently drives this script. 

This patch hunk needs to be removed.

File: install.sh [hunk 2]
```diff
@@ -95,31 +99,34 @@ fi
...
 ask_permission_to_use_homebrew()
 {
-  echo ""
-  echo "Either one or more of the environment variables FC, CC, and CXX are unset or"
-  echo "one or more of the following packages are not in the PATH: pkg-config, realpath, make, fpm."
-  echo "If you grant permission to install prerequisites, you will be prompted before each installation." 
-  echo ""
-  echo "Press 'Enter' to choose the square-bracketed default answer:"
-  printf "Is it ok to use Homebrew to install prerequisite packages? [yes] "
+  cat << EOF | head -c -1
```
Comment (bonachea) [inline install.sh:102]: These `head -c -1` commands are non-POSIX and generate an error on macos

**Review by bonachea — CHANGES_REQUESTED** [review]
@ArthurPV Thanks for the contribution!

Unfortunately these changes include several defects that make it unusable on macOS, at least in its current form. We need the installer to work portably on recent versions of both Linux and macOS.

File: install.sh (hunk 1 above)
Comment (ArthurPV) [reply install.sh:129]: Okay I see, I'll make the change.

**Review by bonachea — APPROVED** [review]
LGTM!

**Pull request merged**